### DISCUSSION TITLE
The SSL request header order is different from the browser, resulting in the target server rejecting access

### DISCUSSION THREAD

**Discussion by 1027663760** [curl/curl#11192] [Ideas]
curl：
:method
:path
:scheme
:authority

browser:
:method
:authority
:scheme
:path

Is there any way to modify this order other than recompiling curl?

**Comment by dfandrich** [comment]
curl sends HTTP-legal requests, so this feature really isn't necessary for standards-compliant operation. IIRC you can move some headers around using `-H`, but not in the general case. Sites are using much more than just header order these days to try to keep scripts away. If that's what you're trying to do, you're probably better off using [curl-impersonate](/lwthiker/curl-impersonate) which does this and more.
